code/game/objects/items/weapons/storage/bible.dm

```diff
@@ -1,3 +1,6 @@
+#define isChaplain(user) (user.mind && user.mind == src.my_rel.religiousLeader)
```

Comment:
What's going on here? The reason why I suggested turning this into a macro was because you didn't need a reference to `src`, but now you do have one... And the two macros are identical? Copy-paste error?

Reply:
I can't into proofreading
I am sorry, thanks for all the review

If I remember correctly, it DID work when I tested it... I'll fix that this evening
Again thank you for helping me

Reply:
Macros including `src` will work fine except when they don't
But they still shouldn't

Reply:
@Exxion you mean when src isn't properly defined ?
Can a macro runtime ?

Reply:
A macro is just a fancy search-and-replace. Whenever you write the left side of the macro in your code, the right side of the macro replaces it, with the arguments replaced. So if you were to call isChaplain with src being something other than a bible, you'd get a "can't find my_rel".